sphericart-torch/src/cuda.cu

```diff
@@ -264,6 +264,8 @@ __global__ void spherical_harmonics_kernel(
          2 * lmax + 1
      );
 
+    int ml = min(static_cast<int>(HARDCODED_LMAX), lmax);
+
```

Comment:
Could this be defined closer to where it is used? There are 100 lines between the variable definition and usage right now

Reply:
I think the whole code needs a bit of internal documentation explaining what it is doing. Can I suggest we do this at a later stage? This is hardly the most confusing bit IMO ^_^